catalog/spatial/geocoding/spatial-geocoding-api/pom.xml

```diff
@@ -28,6 +28,14 @@
             <groupId>ddf.catalog.core</groupId>
             <artifactId>catalog-core-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ddf.catalog.core</groupId>
+            <artifactId>catalog-core-api-impl</artifactId>
```

Comment:
Is there a preference for keeping *-api bundles from depending on implementations at all? I'm curious because it occurred to me to put some shared functionality in this bundle but then I wasn't sure if the aim is to purely interfaces, dependency-inversion style.

Reply:
I think this dependency is just telling Maven it needs to get this code in order to embed it below.

Reply:
But it didn't need to embed it previously, correct?

Reply:
It looks like it's being used to create the AttributeDescriptors. We don't have a service sitting around to provide them, so it looks like the dependency is necessary. :(